packages/core/src/components/fieldset-legend/fieldset-legend.scss

```diff
@@ -0,0 +1,12 @@
+@use '../../helpers';
+
+@mixin FieldsetLegend() {
+  .ods-fieldset-legend {
```

Comment:
How do you feel about `FieldLegend`?

Does it make it seem like it should be a children of `Field`'s instead?

Reply:
Yes, as this is only used with `Fieldset` but should not be used with `Field`.

I think complexity will be if we want to do some functionality on `Field` that is to be also on `Fieldset`, but it's then another thought on how to share and reuse the code.